### PR TITLE
Rw feature schema

### PR DESCRIPTION
**Pull request by ringo-w**
Implement Schema

### CODE REVIEW

File: server/datamodel.prisma [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+  favourite: [String]
 }
+type Campsite {
+  id: ID! @id
+  campname: String!
+  price: Int!
+  description: String
+  facilities: [String]
+  details: [String]
+  reviews: [String]
```
Comment (siddharthparmar7) [inline server/datamodel.prisma]: Review should be a separate node with id, comment and who wrote it for which campsite.

File: server/datamodel.prisma [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
+  location: [Float!]!
```
Comment (siddharthparmar7) [inline server/datamodel.prisma]: should be lat, long?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the changes.

File: server/datamodel.prisma (hunk 2 above)
Comment (ringo-w) [reply server/datamodel.prisma]: It was going to be an array of two indexes for lat and long but I could separate it.  Which is better?

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**